.buildkite/hooks/post-command

```diff
@@ -1,3 +1,7 @@
 #!/usr/bin/env bash
 
-.buildkite/scripts/lifecycle/post_command.sh
+if [ "$SKIP_KIBANA_HOOKS" = "1" ]; then
```

Comment:
Sometimes, these pipeline files will be triggered from non-kibana-buildkite agents, where these hooks are not necessary (because the jobs just trigger downstream) and also would fail (because the agents aren't prepared the same way).